aemedge/blocks/marquee/marquee.js

```diff
@@ -92,10 +92,14 @@ function setupBGPictures(block) {
 
 // read the config and construct the DOM
 function processBlockConfig(block) {
-  const marqueContent = createTag('div', { class: 'marquee-content' });
   const mediaDIV = createTag('div', { class: 'foreground-container' });
   const nonMediaDIV = createTag('div', { class: 'text-cta-container' });
   const btnsDIV = createTag('div', { class: 'buttons-container' });
+  const shouldAddGradient = true; // or 'true'
```

Comment:
You have this set to true every where so it is being added no matter what.
The page in your PR before/after does not have this value configured, so I don't expect to see the class.
![image](https://github.com/user-attachments/assets/aef8b11e-ea14-4b7f-8f47-019c48a6ed81)